python_modules/dagster/dagster/_core/storage/sql.py

```diff
@@ -219,3 +219,11 @@ def db_select(items: Iterable):
         return db.select(*items)
 
     return db.select(items)
+
+
+def db_subquery(query, name: str = "subquery"):
+    """Utility class that allows compatibility between SqlAlchemy 1.3.x, 1.4.x, and 2.x."""
+    if db.__version__.startswith("2."):
```

Comment:
thoughts on making this not startswith("1.") instead? so it doesn't break someday on 3.*?